tw-NEDa-frontend/src/app/components/ong/ong.component.html

```diff
@@ -0,0 +1,175 @@
+            <h2 class="panel-title text-center">
+              Search by ONG Name
+            </h2>
+          </div>
+
+          <!--panel body-->
+          <div class="panel-body">
+            <form name="earthquakeForm" novalidate>
+              <div class="form-group">
+                <input
+
+                  (keyup)='checkMagnitude($event)'
+                  placeholder="NGO name"
+                  type="text">
+                <button md-button>Search</button>
+              </div>
+            </form>
+
+
+          </div>
+
+        </div>
+
+
+        <!--LOCATION SEARCH -->
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <!--title form-->
+            <h2 class="panel-title text-center">
+              Search by Location
+            </h2>
+          </div>
+
+          <!--panel body-->
+          <div class="panel-body">
+            <form name="earthquakeForm" novalidate>
+              <div class="form-group">
+                <input
+
+                  (keyup)='checkMagnitude($event)'
+                  placeholder="Location"
+                  type="text">
+                <button md-button>Search</button>
+              </div>
+            </form>
+
+
+          </div>
+
+        </div>
+
+
+        <!--ACTIVITY SEARCH -->
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <!--title form-->
+            <h2 class="panel-title text-center">
+              Search by Activty
+            </h2>
+          </div>
+
+          <!--panel body-->
+          <div class="panel-body">
+            <form name="earthquakeForm" novalidate>
+              <div class="form-group">
+                <input
+
+                  (keyup)='checkMagnitude($event)'
+                  placeholder="Activity"
+                  type="text">
+                <button md-button>Search</button>
+              </div>
+            </form>
+
+
+          </div>
+
+        </div>
+
+
+      </div>
+
+
+      <div class="col-md-8 table-responsive">
+
+
+        <div class="tttable">
+          <ngx-datatable
+            id="eq-table"
+            #table
+            class='material'
+            [columns]="columns"
+            [columnMode]="'force'"
+            [footerHeight]="50"
+            [rowHeight]="'auto'"
+            (rowContextmenu)="onContextMenu($event)"
+            [limit]="10"
+            [rows]='rows'>
+          </ngx-datatable>
+
+          <template #hdrTpl>
+
+          </template>
+          <template #editTmpl let-row="row" let-value="value" let-i="index">
+            <button md-icon-button [mdMenuTriggerFor]="rowMenu">
+              <md-icon>more_vert</md-icon>
+            </button>
+
+            <md-menu #rowMenu="mdMenu">
+              <button md-menu-item (click)="edit(row)">
+                <md-icon>edit</md-icon>
+                Edit
+              </button>
+
+              <button md-menu-item (click)="delete(row)">
```

Comment:
I don't think is good that the user will have the option to delete a row or to add an earthquake.We should keep only the filter card where we give him the option to see the table in which way he wants.